### PR TITLE
Lunar Hopper Tutorial update

### PR DESCRIPTION
**Pull request by gtep96**
Requesting review

### CODE REVIEW

File: docs_tutorials/lunar_hopper/lunar_hopper.md [hunk 1]
```diff
@@ -5,241 +5,149 @@ sidebar_label: 'Lunar Hopper'
...
+## CITROS Initialization
 
+1. [Install CITROS](https://citros.io/doc/docs_tutorials/getting_started/#installation).
+2. Follow [these steps](https://citros.io/doc/docs_tutorials/getting_started/#initialization) to Initialize CITROS.
 
+Now you can see .citros directory in the explorer.
 
-:::tip
+## Scenario
+For this tutorial, let's check how far the Hopper can hop depending on Specific Impulse. The optimal trajectory computes by solving non-linear optimal control problems(OCP) in the standard Bolza form using pseudo-spectral collocation methods and adjusted using an additional real dynamic function. The OCP solver used in this example is MPOPT (based on IPOPT) library modified by Lulav Space team. You can find more information about MPOPT optimal control solving library on the MPOPT GitHub or website.<br/>
```
Comment (mariaBer) [inline docs_tutorials/lunar_hopper/lunar_hopper.md]: more information about MPOPT optimal control solving library on the MPOPT GitHub or website.<br/> 
link to github and/or website

Comment (mariaBer) [inline docs_tutorials/lunar_hopper/lunar_hopper.md]: For this tutorial, let's check how far the Hopper can hop depending on Specific Impulse.
remove

**Review by mariaBer — CHANGES_REQUESTED** [review]

File: docs_tutorials/lunar_hopper/lunar_hopper.md (hunk 1 above)
Comment (gtep96) [reply docs_tutorials/lunar_hopper/lunar_hopper.md]: Done

Comment (gtep96) [reply docs_tutorials/lunar_hopper/lunar_hopper.md]: Done

File: docs_tutorials/lunar_hopper/lunar_hopper.md [hunk 2]
```diff
@@ -5,241 +5,149 @@ sidebar_label: 'Lunar Hopper'
...
+The main goal of the project is to find the optimal way to "hop" on the Moon as far as possible with given vessel parameters.<br/>
```
[the remark this answers is not in the document]
Comment (gtep96) [reply docs_tutorials/lunar_hopper/lunar_hopper.md:72]: Done

File: docs_tutorials/lunar_hopper/lunar_hopper.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gtep96) [reply docs_tutorials/lunar_hopper/lunar_hopper.md]: Done

**Comment by gtep96** [comment]
@mariaBer updated and pushed

File: docs_tutorials/lunar_hopper/lunar_hopper.md [hunk 3]
```diff
@@ -3,243 +3,158 @@ sidebar_position: 60
...
-Navigate to our ```Code``` project page, open the Notebooks folder and click on the notebook file. Here you can see the usual Jupiter editor interface: you can add blocks of code or built-in Markdown engine, run and save notebook and control the Python kernel.
+Learn more about parameter setup and defining custom functions in [Directory parameter_setups](https://citros.io/doc/docs_cli/structure/citros_structure/#directory-parameter_setups) and [Adding Functions to Parameter Setup](https://citros.io/doc/docs_cli/configuration/config_params) pages.
 
-You can find all the data analysts package guides and API reference [here](https://citros.io/doc/docs_data_analysis).
+In addition to parameter setup, you can configure the simulation performance setup (timeout, CPU, GPU and Memory) as well.
+This parameters can be found in ```.citros/simulations/simulation_lunar_hopper.json```. <br/>
+The default setup is 600 seconds timeout, 4 CPU, 4 GPU and 4096 MB of Memory.
```
Comment (mariaBer) [inline docs_tutorials/lunar_hopper/lunar_hopper.md:111]: Am I missing something?
image

File: docs_tutorials/lunar_hopper/lunar_hopper.md [hunk 4]
```diff
@@ -3,243 +3,158 @@ sidebar_position: 60
...
++ running batch [<batch_run / batch name>], description: local test run, repeating simulations: [1]
++ + running simulation [0]
+...
+```
 
-Run the ```Lunar_hopper``` simulation and copy your batch id (we will need it later).
+All the results will be saved under `.citros/runs/[simulation_name]` folder.
 
-Let's return to our Notebook and check the code: to start with, we need to import all the necessary modules:
+To plot the local run results you can use [Foxglove](https://citros.io/doc/docs_tutorials/#visualization-with-foxglove).
```
Comment (mariaBer) [inline docs_tutorials/lunar_hopper/lunar_hopper.md:136]: 1. Should both of the layouts work? should one? add the relevant information
2. None of the layouts worked for me

**Review by mariaBer — CHANGES_REQUESTED** [review]

File: docs_tutorials/lunar_hopper/lunar_hopper.md (hunk 3 above)
Comment (gtep96) [reply docs_tutorials/lunar_hopper/lunar_hopper.md:111]: @mariaBer https://github.com/lulav/citros_doc/pull/101#discussion_r1431885192

**Comment by gtep96** [comment]
updated and pushed

File: docs_tutorials/lunar_hopper/lunar_hopper.md (hunk 4 above)
Comment (mariaBer) [reply docs_tutorials/lunar_hopper/lunar_hopper.md:147]: If you mention foxglove, add one of the results foxglove gives

**Comment by gtep96** [comment]
updated and pushed

**Pull request merged**